commands.go

```diff
@@ -114,6 +117,31 @@ func (b *bot) sudoku(m dggchat.Message, s *dggchat.Session) {
 	s.SendMute(m.Sender.Nick, -1)
 }
 
+func (b *bot) frenchToastAlert(m dggchat.Message, s *dggchat.Session) {
+	type ftlXML struct {
+		XMLName xml.Name `xml:"frenchtoast"`
+		Status  string   `xml:"status"`
+	}
```

Comment:
```suggestion
	f := struct {
		XMLName xml.Name `xml:"frenchtoast"`
		Status  string   `xml:"status"`
	}{}
```

Reply:
Yeah I only made this as a meme for like 2 chatters (it is a regional new england thing). 
But I could look into making it into a command to see if you should go out and get groceries, before a large amount of snow is about to hit you.